packages/notification-center/src/i18n/languages/sm.ts

```diff
@@ -0,0 +1,11 @@
+import { ITranslationEntry } from '../lang';
+
+export const SI: ITranslationEntry = {
+  translations: {
+    notifications: 'faasilasilaga',
+    markAllAsRead: 'faailoga Mea uma e pei ona Faitau',
+    poweredBy: 'fa amalosia ',
+    settings: 'faatulagaga',
+  },
+  lang: 'sm',
```

Comment:
Inverting Google translation (as I don't know samoan) return some funny translations. Are you sure these are accurate translations?

Reply:
@P-fernandez
I am 99 percent sure that this is the correct translation becuase one of my friend taught me this language so I know it very well.
and about Google translate, it's not that accurate so we shouldn't always believe on it.
Rest up to you :)

Reply:
Also be assured, if I won't be sure or I won't know this language well, I wouldn't make a PR for it, and I never believe on Google Cloud.
I am also learning French but sometimes it shows wrong translations